Reference/Routing/WebApi/authorization.md

```diff
@@ -44,12 +43,9 @@ To secure your controller based on front-end membership use the attribute: `Umbr
 
 *It's important to note the namespace since we have another class called MemberAuthorizeAttribute in a different namespace that is used for MVC.*
 
-There are 4 parameters that can be supplied to control how the authorization works:
+There are 3 parameters that can be supplied to control how the authorization works:
 
 ```csharp
-// Flag for whether to allow all site visitors or just authenticated members
-// This is the same as applying the [AllowAnonymous] attribute
-bool AllowAll
```

Comment:
Did the `AllowAll` parameter never exist or has it been removed in v8? 🤔 If it still exists in v7 we should probably have this article versioned 🙂

Reply:
I checked in the v7 branch of code and although it was available, it was obsolete in favour of [AllowAnonymous].

I am happy to create a versioned one if this is preferred approach :)

Reply:
No that's perfectly fine then - thanks a lot! 🙂🎉